students/barecki_andrzej/lesson_09_reading_and_writing_files/mad_libs.py

```diff
@@ -0,0 +1,14 @@
+word_keys = ["ADJECTIVE", "NOUN", "ADVERB", "VERB"]
```

Comment:
to jest stała, więc zgodnie z PEP8 należałoby ją nazwać wielkimi literami:
`WORD_KEYS = [...]`